src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -50,6 +50,9 @@ function LtiConfigForm({
       <Form ref={formRef} onSubmit={handleSubmit}>
         <h3 className="mb-3">{title}</h3>
         <p>{intl.formatMessage(messages.formInstructions)}</p>
+        {app.messages && Object.values(app.messages).flat().map(msg => (
```

Comment:
Just a note, as this PR depends upon the backend pr, I'll take a look once that PR is merged.

Reply:
@kewne After the change in the backend, we now have a list `app.messages`. We can simplify this statement now.

`app.messages.map(....)`

Reply:
Fixed!